packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/styles.ts

```diff
@@ -1,9 +1,9 @@
-export const sidebar = (width: number) =>
+export const sidebar = (width: number, sidebarOffset: boolean) =>
```

Comment:
in this case, `sidebarOffset` would be of type `number`

Reply:
updated to type `number`, thanks for catching that